cpp/src/parquet/schema.cc

```diff
@@ -28,6 +28,7 @@
 #include "parquet/exception.h"
 #include "parquet/schema_internal.h"
 #include "parquet/thrift_internal.h"
+#include "parquet/windows_fixup.h"  // for OPTIONAL
```

Comment:
I had to add the windows fixup here to overcome 

```
In file included from D:/a/arrow/arrow/build/cpp/src/parquet/CMakeFiles/parquet_shared.dir/Unity/unity_3_cxx.cxx:16:
D:/a/arrow/arrow/cpp/src/parquet/schema.cc: In function 'void parquet::schema::PrintRepLevel(parquet::Repetition::type, std::ostream&)':
D:/a/arrow/arrow/cpp/src/parquet/schema.cc:630:30: error: expected unqualified-id before ':' token
  630 |     case Repetition::OPTIONAL:
```

https://github.com/apache/arrow/actions/runs/14115826826/job/39545693922?pr=45360#step:7:838